lib/hw_unique_key/Kconfig

```diff
@@ -4,6 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+config HW_UNIQUE_KEY_LOAD
+	bool "Load Hardware Unique key on boot"
+	depends on HAS_HW_NRF_ACL
+	depends on HAS_HW_NRF_CC310
+	depends on NRF_CC3XX_PLATFORM
+	select FPROTECT
+	imply FPROTECT_ENABLE_NO_ACCESS
+	imply NRFX_NVMC
```

Comment:
Should we add some help text here explaining that the bootloader will expect the app to write a HUK upon first boot?

Reply:
Added some text